plugin.video.iplayerwww/resources/settings.xml

```diff
@@ -0,0 +1,55 @@
+<settings>
```

Comment:
would be nice if at some point you can convert the settings to the new format. You can use https://gitlab.com/ronie/script.settings-convert to automatically convert them for you

Reply:
I will wait for the krypton branch to come to an end before starting with that. Would like to keep code as close as possible.